cr3qt/data/docx.css

```diff
@@ -50,19 +50,23 @@ ol { display: block; list-style-type: decimal; margin-left: 1em }
 .h4 { font-size: 120% }
 .h5 { font-size: 110% }
 
-table { font-size: 80% }
+table { font-size: 80%; width: 100%; }
 td, th { text-indent: 0px; padding: 3px }
-th {  font-weight: bold; text-align: center; background-color: #DDD  }
+/* table borders */
+table, th, td { border-width: 2px; border-style: ridge; border-color: #888; border-collapse: collapse; }
```

Comment:
This is in order to be consistent with tables in the EPUB style? I'd normally expect a simple solid border in DOCX.

Reply:
I just copied it from fb2.css without any further thought :) The proper way would be to read border properties for the docs  but I'm not going to do it at least in a foreseeable future. So I'll leave it as is for now and let others decide how should it be.